src/xcode/ENA/ENA/Source/Models/AppInformation/AppInformationDetailModel.swift

```diff
@@ -0,0 +1,38 @@
+// Corona-Warn-App
```

Comment:
This file was deleted and should not be re-added again.

Reply:
Also removed